content/en/blog/2019/announcing-1.3/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Istio 1.3
+subtitle: Major Update
+description: Istio 1.3 release announcement.
+publishdate: 2019-09-10
+attribution: The Istio Team
+release: 1.3.0
+---
+
+We are pleased to announce the release of Istio 1.3!
+
+{{< relnote linktonote="true" >}}
+
+The theme of 1.3 is User Experience - improving the experience of new users adopting Istio, debugging problems and supporting more applications out of the box. 
+
+Users are adopting Istio faster than ever before and we’ve been listening to the feedback! We’ve heard that Istio can be complex and challenging to adopt and we've been working hard to make improvements. 
+
+We started by forming a new User Experience team earlier this year to lead improvements across all components of Istio. Everyone is invited to join the working group meetings and share your thoughts. We’ve begun delivering many notable improvements, some of which are ready for you to use in 1.3 and some in experimental state. 
+
+In the past, we required you to explicitly declare the protocol for service ports. This was a common gotcha as users forgot to name their ports when adding their application to the mesh. By default, Pilot now has protocol sniffing which removes this requirement! Along the same lines, another obsolete requirement is the need to define all the pod ports (`containerPort`) for each container. Istio will now capture all inbound ports by default.
+
+If you haven't tried out `istioctl` yet, now is a great time to start! Formerly experimental sub-commands `metrics`, `dashboard` and `convert-ingress` have been promoted and are now supported Istio features. In addition, several new istioctl experimental subcommands have been added to help improve the usability of the system.
+
+The Istio team is also constantly working on improving the control plane performance and reducing its footprint. In this release, Pilot performance has been significantly improved resulting in as much as a 90% CPU usage savings!
+
+Locality aware Load Balancing is another feature which graduated from experimental to default in this release. Istio can now take advantage of existing locality information to prioritize load balancing pools and favor sending requests to the closest backends.
+
+Control plane monitoring has been greatly enhanced with additional metrics to monitor configuration state, Sidecar Injector webhook, a new Grafana dashboard for Citadel, and improvements to the Pilot dashboard.
+
```

Comment:
Could we add a paragraph and a list highlighting some of the work of the docs team to share the biggest pieces we crafted for this release? @geeknoid @sdake @frankbu WDYT? I've added the two biggest items I could think of in my suggestion, would you include other highlights?

```suggestion
We've heard how important it is for you to have accurate and up-to-date information in our documentation. The documentation team continuously works to provide with the information you need to be successful. In Istio 1.3, we made many improvements but the following are some highlights:

- Added the new [Istio Deployment Models concept](/docs/concepts/deployment-models/) to help you decide what deployment model suits your needs.
- Organized the content in of our [Operations Guide](/docs/ops/) and created a [section with all troubleshooting tasks](/docs/ops/troubleshooting) to help you find the information you seek faster. 

Thanks to our community testers, our [documentation](/docs/) is tested for every release including Istio 1.3!
```